arb_os/evmOps.mini

```diff
@@ -350,11 +338,15 @@ public impure func evmOp_extcodehash(addrAsUint: uint) -> bytes32 {
         if (account_isEmpty(acct)) {
             return bytes32(0);
         }
-        if let Some(code) = account_getCode(acct) {
-            return keccak256(code, 0, bytearray_size(code));
+        return xif (evmCallFrame_constructorIsOnStackForAddress(topFrame, addr)) {
+            bytes32(0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470)  // hash of empty code
         } else {
-            return bytes32(0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470);
-        }
+            xif let Some(code) = account_getCode(acct) {
+                keccak256(code, 0, bytearray_size(code))
+            } else {
+                bytes32(0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470)  // hash of empty code
```

Comment:
I believe mini has constants, now that this is used twice, should this be moved to a constant?

Reply:
Done.